packages/composables/src/useNotifications.ts

```diff
@@ -93,7 +93,7 @@ export function useNotifications(): UseNotificationsReturn {
     const messageId = geterateId();
     _notifications.value.push({
       id: messageId,
-      type: options.type || "info",
+      type: options.type as NotificationType,
```

Comment:
this casting is incorrect. `type` is optional so we had `info` as default type, now it would send `undefined as NotificationType`

Reply:
okay, so then just typing needs to change so we won't need to do casting, so this internal method needs to require `options.type`

Reply:
done